PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+## Please give the PR a descriptive title.
+
+## Type of change
+
+- [ ] Bug Fix
+- [ ] New Feature
+- [ ] Refactor
+- [ ] Project Build
+- [ ] Project Configuration
+
+## Submission Checklist
+
+- [ ] Have you followed the guidelines in our Contributing document?
+- [ ] Did you create appropriate tests?
+
+## Description
+
+A few sentences describing the overall goals of the pull request's commits.
+
+## Related Issues / Bugs / Feature Requests
+
+Link to any appropriate GitHub/Jira issues, bugs or freature requests
```

Comment:
You can link to issues/prs if you start typing `#`:
e.g. `#39`

Maybe it's common knowledge?

Reply:
Yes!